ACE/ace/SOCK_Dgram.cpp

```diff
@@ -746,31 +798,33 @@ ACE_SOCK_Dgram::make_multicast_ifaddr6 (ipv6_mreq *ret_mreq,
       // Initial call to determine actual memory size needed
       DWORD dwRetVal;
       ULONG bufLen = 0;
+      char *buf;
```

Comment:
initialise with 0, then the = 0 on line 808 can go away

Reply:
And then I can invert the condition. Gotcha.